docs/artifacts_test.md

```diff
@@ -32,7 +32,7 @@ jenkins-pipelines
 You should use different parameters for .rpm/.deb tests and for AMI test:
 - The only required parameter for .rpm/.deb jobs is `scylla_repo`, a path to a ScyllaDB repo (e.g., `https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/master/latest/scylla.repo`)
 - For AMI job you need two parameters: `scylla_ami_id` and `region_name`
-- For Docker job you need two parameters: `scylla_docker_image` and `scylla_version`
+- For Docker job you need three parameters: `scylla_docker_image` (e.g., `scylladb/scylla` or `scylladb/scylla-nightly`), `scylla_version` (it'll be used as Docker image tag) and `scylla_repo`
```

Comment:
Why do we need `scylla_repo` ? for downloading debug symbols ?, I would argue we don't need it in artifact test.

And if we really need it, and it not configured in advance in the docker itself then we should look it up.

Reply:
Removed this requirement. Not sure if we need to run SCT against bare OS as a Docker container, but for now it's not supported anyway.